tests/merger_test.py

```diff
@@ -81,7 +80,7 @@ def test_merge_point_data_unique_entity(tmp_path):
         data.append(
```

Comment:
The first test functio is not pertinent anymore if we force data to have same name and entity type. 
Remove this one